proto/rill/admin/v1/api.proto

```diff
@@ -1138,17 +1162,19 @@ enum GithubPermission {
   GITHUB_PERMISSION_WRITE = 2;
 }
 
-message GetGitCredentialsRequest {
+message GetArtifactsURLRequest {
   string organization = 1 [(validate.rules).string.min_len = 1];
   string project = 2 [(validate.rules).string.min_len = 1];
 }
 
-message GetGitCredentialsResponse {
+message GetArtifactsURLResponse {
   string repo_url = 1;
   string username = 2;
   string password = 3;
   string subpath = 4;
   string prod_branch = 5;
+  // either upload_path or git related details will be set
+  string upload_path = 6;
 }
```

Comment:
Actually I was passing complete path instead of generating a `download_url`. Since this is for internal users only who will have access to gcs buckets I thought we can just pass path and they can use `gcloud cli` to copy contents. 
I can also change it to `download_url` if that is preferable.

Reply:
Since we already have logic for a signed download URL, I think that would be nice. Makes it more generic and the returned URL can be used with `curl` from all devices.